.github/workflows/github-actions.yml

```
@@ -0,0 +1,171 @@
+name: CI/CD with github actions & docker
+
+on:
+  push:
+    branches: [ "develop" ]
```

```suggestion
    branches: [ "main" ]
```
음 지금 저희가 main까지 머지를 안하고 있긴 한데, 실 서비스는 main 기준으로 배포가 돼야될것 같아요!

---

추가 완료 했습니다 🫡